package.json

```diff
@@ -12,6 +12,6 @@
   },
   "homepage": "https://github.com/galaxyproject/training-material#readme",
   "dependencies": {
-    "decktape": "^2.8.3"
+    "decktape": "^2.8.9"
```

Comment:
Can you rebase this commit excluding the changes to this file and `package-lock.json` ?